src/test/kotlin/domain/LottoStoreTest.kt

```diff
@@ -1,23 +1,57 @@
 package domain
 
+import org.assertj.core.api.Assertions
 import org.assertj.core.api.Assertions.assertThat
-import org.assertj.core.api.Assertions.assertThatIllegalArgumentException
-import org.junit.jupiter.params.ParameterizedTest
-import org.junit.jupiter.params.provider.ValueSource
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertAll
 
 class LottoStoreTest {
-    @ValueSource(ints = [1000, 100000])
-    @ParameterizedTest
-    fun `구매금액이 1000원 이상 100001원 이하면 개수만큼 로또를 반환할 수 있다`(amount: Int) {
-        val lotto: List<Lotto> = LottoStore(RandomLottoGenerator).buyLotto(amount)
-        assertThat(lotto.size).isEqualTo(amount / 1000)
+    private val lottoStore = LottoStore(RandomLottoGenerator)
+
+    inner class TestLottoGenerator : LottoGenerator {
+        private var lotateTickets = listOf(0, 1, 2, 3, 4, 5)
+        override fun generateLotto(): Lotto {
+            lotateTickets = lotateTickets.map { (it + 1) % 45 }
+            return Lotto(*lotateTickets.toIntArray())
+        }
+    }
+
+    @Test
+    fun `원하는 번호들의 수동 로또들을 구매한다`() {
+        val money = Money(2000)
+        val lottoTickets: LottoTickets = lottoStore.buyManualLotto(money.count, intArrayOf(1, 2, 3, 4, 5, 6), intArrayOf(1, 11, 22, 33, 44, 45))
+        assertAll({
+            assertThat(lottoTickets.size).isEqualTo(2)
+            assertThat(lottoTickets[0].toList().map { it.toInt() }).isEqualTo(listOf(1, 2, 3, 4, 5, 6))
+            assertThat(lottoTickets[1].toList().map { it.toInt() }).isEqualTo(listOf(1, 11, 22, 33, 44, 45))
+        })
```

Comment:
이렇게 각 검증문들의 람다를 넘겨주면 됩니다 :)
```suggestion
        assertAll(
            { assertThat(lottoTickets.size).isEqualTo(2) },
            { assertThat(lottoTickets[0].toList().map { it.toInt() }).isEqualTo(listOf(1, 2, 3, 4, 5, 6)) },
            { assertThat(lottoTickets[1].toList().map { it.toInt() }).isEqualTo(listOf(1, 11, 22, 33, 44, 45)) },
        )
```

Reply:
방금 함수 정의 부분 열어보고 vararg을 전달하는 것으로 해당 내용에 대해서 이해하였습니다!
```kotlin
fun assertAll(vararg executables: () -> Unit) =
    assertAll(executables.toList().stream())
```
지금 제 코드처럼 하게 된다면 assertAll의 역할을 수행하지 못한다고 이해할 수 있었습니다!
알려주셔서 감사합니다!